src/Features/CSharp/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -180,6 +185,81 @@ private bool CanUseVar(ITypeSymbol typeSymbol)
             return Tuple.Create(newSemanticDocument, newMatches);
         }
 
+        private Document RewriteExpressionBodiedMemberAndIntroduceLocalDeclaration(
+            SemanticDocument document,
+            ExpressionSyntax expression,
+            NameSyntax newLocalName,
+            LocalDeclarationStatementSyntax declarationStatement,
+            bool allOccurrences,
+            CancellationToken cancellationToken)
+        {
+            var oldBody = expression.GetAncestorOrThis<ArrowExpressionClauseSyntax>();
+            var oldParentingNode = oldBody.Parent;
+
+            var newStatement = Rewrite(document, expression, newLocalName, document, oldBody.Expression, allOccurrences, cancellationToken);
+            var newBody = SyntaxFactory.Block(declarationStatement, SyntaxFactory.ReturnStatement(newStatement))
+                                       .WithLeadingTrivia(oldBody.GetLeadingTrivia())
+                                       .WithTrailingTrivia(oldBody.GetTrailingTrivia())
+                                       .WithAdditionalAnnotations(Formatter.Annotation);
+
+            SyntaxNode newParentingNode = null;
+            if (oldParentingNode is BasePropertyDeclarationSyntax)
+            {
+                var getAccessor = SyntaxFactory.AccessorDeclaration(SyntaxKind.GetAccessorDeclaration, newBody);
+                var accessorList = SyntaxFactory.AccessorList(SyntaxFactory.List(new AccessorDeclarationSyntax[] { getAccessor }));
+
+                newParentingNode = ((BasePropertyDeclarationSyntax)oldParentingNode).RemoveNode(oldBody, SyntaxRemoveOptions.KeepNoTrivia);
```

Comment:
I notice I have a test for trailingtrivia but not for leadingtrivia. Let me add an unit test to make sure trivia is kept.